be/src/vec/exec/format/parquet/vparquet_reader.cpp

```diff
@@ -788,7 +788,7 @@ Status ParquetReader::_process_page_index(const tparquet::RowGroup& row_group,
             // use the union row range
             skipped_row_ranges.emplace_back(skipped_row_range);
         }
-        _col_offsets.emplace(parquet_col_id, offset_index);
+        _col_offsets.emplace(read_col, offset_index);
```

Comment:
Only columns in `ColumnValueRanges` can reach here, and should understand  what `parquet_col_id` stands for.